mne/decoding/tests/test_time_gen.py

```diff
@@ -235,6 +236,12 @@ def test_generalization_across_time():
     gat.predict(epochs)
     assert_raises(ValueError, gat.predict, epochs[:10])
 
+    # Check that still works with classifier that output y_pred with
+    # shape = (n_trials, 1) instead of (n_trials,)
+    gat = GeneralizationAcrossTime(clf=RANSACRegressor(LinearRegression()))
+    gat.fit(epochs)
```

Comment:
how long does it take to run? I would use as little number of epochs as possible.